stdlib/jms/src/main/ballerina/jms/simple_durable_topic_subscriber.bal

```diff
@@ -101,6 +101,7 @@ public type SimpleDurableTopicSubscriber object {
     # Creates a text message that can be sent through any JMS message producer to a queue or topic.
     #
     # + message - text content of the message
+    # + return - Returns a message or nill if the session is nil
```

Comment:
should we say something like "Returns the created message, or nil if the session is nil"?